platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -111,7 +112,86 @@ public void testIsJavaIdentifier() throws Exception {
         assertFalse(BaseUtilities.isJavaIdentifier("some.thing"));
     }
 
+    //--------------------------------------------------------------------------
+    public void test_toFile_throwsNullPonter_whenArgumentIsNull()
+            throws Exception {
+
+        try {
+            toFile(null);
+            fail();
+        } catch (final NullPointerException e) {
+            return;
+        }
+    }
+
+    //--------------------------------------------------------------------------
+    public void test_toFile_returnsFile_whenGivenUnixPath()
```

Comment:
Is this test already included in some of the other tests? Why is is neccessary?

Reply:
> The new tets methods are intended to replase the "testFileURI" test. I intend to remove "testFileURI" in next PR after succesful nightly build (so that there is proof that both old and new test pass).

good thinking but there is no need to wait. PRs should be somewhat standalone if possible. Look at it this way:
 - we know that the old tests are green before this PR
 - this PR is already built and tested (I just unlocked the gh workflow here for another set of tests) and will be automatically tested again post merge (-> there already is a documented before and after)
 - so feel free to remove the old test case which is supposed to be replaced in the same PR you add the replacement cases